.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,412 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="LINE_SEPARATOR" value="&#10;" />
+    <option name="RIGHT_MARGIN" value="150" />
+    <AndroidXmlCodeStyleSettings>
+      <option name="USE_CUSTOM_SETTINGS" value="true" />
+    </AndroidXmlCodeStyleSettings>
+    <JavaCodeStyleSettings>
+      <option name="FIELD_NAME_PREFIX" value="m" />
```

Comment:
How does Android Studio check this when using reformat code? Does it suggest you to add `m` character if is not there?

Reply:
It's this
![image](https://user-images.githubusercontent.com/3314607/51993616-82921800-24af-11e9-9dc9-f480208811fb.png)

Reply:
You get no lint warning, when you don't have it.
In my point of view, it's not that important, because the future is Kotlin and don't suggest it